### ISSUE TITLE
FQTN displayed as "Non | demo-tests/test_1.py::test_5_marked_SKIP" for demo test

### ISSUE THREAD

**Issue by jeffwright13** [jeffwright13/pytest-tui#70]
No `start_time` assigned to this test somehow?

Test 'test_1.py::test_5_marked_SKIP' source code:
```
@pytest.mark.skip
def test_5_marked_SKIP(log_testname):
    logger.critical(fake_data())
    logger.error(fake_data())
    logger.warning(fake_data())
    logger.info(fake_data())
    logger.debug(fake_data())
    assert 1
```

TuiTestResult object for this test:
```
TuiTestResult(fqtn='demo-tests/test_1.py::test_5_marked_SKIP', outcome='SKIPPED', start_time=None, duration=0.0006051778327673674, caplog='', capstderr='', capstdout='', longreprtext="('/Users/jwr003/coding/pytest-tui/demo-tests/test_1.py', 187, 'Skipped: unconditional skip')\n").to_dict()
```
As a dictionary:
```
{'fqtn': 'demo-tests/test_1.py...arked_SKIP', 'outcome': 'SKIPPED', 'start_time': None, 'duration': 0.0006051778327673674, 'caplog': '', 'capstderr': '', 'capstdout': '', 'longreprtext': "(...)\n"}
special variables
function variables
'fqtn': 'demo-tests/test_1.py::test_5_marked_SKIP'
'outcome': 'SKIPPED'
'start_time': None
'duration': 0.0006051778327673674
'caplog': ''
'capstderr': ''
'capstdout': ''
'longreprtext': "('/Users/jwr003/coding/pytest-tui/demo-tests/test_1.py', 187, 'Skipped: unconditional skip')\n"
len(): 8
```

**Comment by jeffwright13** [comment]
Apparently, `pytest_runtest_setup` doesn't even get called for this test (which is where the `start_time` values gets populated). It has to be because of the `@pytest.mark.skip` marker. This is the only test in the suite that has that marker, and it's the only one skipping the start-time-populating code.

**Comment by jeffwright13** [comment]
Fixed in 1.7.0
